plugins/storage/volume/scaleio/src/main/java/org/apache/cloudstack/storage/datastore/util/ScaleIOUtil.java

```diff
@@ -53,6 +54,16 @@ public class ScaleIOUtil {
     // VOL-ID 6c33633100000009 MDM-ID 218ce1797566a00f
     // VOL-ID 6c3362a30000000a MDM-ID 218ce1797566a00f
 
+    private static final String QUERY_GUID_CMD = "drv_cfg --query_guid";
+    // Sample output for cmd: drv_cfg --query_guid:
+    // B0E3BFB8-C20B-43BF-93C8-13339E85AA50
```

Comment:
is this javadoc for the above constant?

Reply:
> is this javadoc for the above constant?

no, just to indicate sample output for the cmd.

Reply:
Can we move this to some javadoc? it is a bit out of the blue.

Reply:
ok @DaanHoogland